pyproject.toml

```diff
@@ -17,12 +17,17 @@ classifiers = [
 python = ">=3.9"
 
 [tool.poetry.group.dev.dependencies]
-pytest = "^8.3.2"
-ruff = "^0.6.3"
```

Comment:
I'd prefer not to remove `ruff` since I don't like waiting for pre-commit to do formatting during dev.

Reply:
Re-ruffed.